app/views/bourbons/index.html.erb

```diff
@@ -1,16 +1,42 @@
-<h1>List of Bourbons</h1>
-<ul>
-<% @bourbons.each do |bourbon| %>
-  <li>
-    <%= link_to bourbon.name, bourbon_path(bourbon)%> - <%=bourbon.proof%> <%=bourbon.distillery%> - <%=bourbon.varietal%>
-  </li>
-<% end %>
-</ul>
-
 <% if current_user %>
   <%= link_to "Add Bourbon", new_bourbon_path %>
 <% end %>
 
 <%= link_to "Sign Up", new_user_registration_path %>
 
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <% if current_user && current_user.admin? %>
+      <th>Update</th>
+      <th>Delete</th>
+      <% end %>
+      <th>Proof</th>
+      <th>Expert</th>
+      <th>User</th>
+      <th class="distillery">Distillery</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @bourbons.each do |bourbon| %>
+    <tr>
+      <td><%= link_to bourbon.name, bourbon %></td>
+      <% if current_user && current_user.admin? %>
+      <td>
+        <%= link_to 'Update', edit_bourbon_path(bourbon), class: 'button tiny' %>
+      </td>
+      <td>
+        <%= button_to 'Delete', bourbon_path(bourbon), method: :delete %>
+      </td>
+      <% end %>
+      <td><%= bourbon.proof %></td>
+      <td>92</td>
+      <td>96</td>
+      <td class="distillery"><%= bourbon.distillery %></td>
+    </tr>
+    <% end %>
```

Comment:
Your indentation through this whole page needs fixing - pay attention to your `if/else` blocks and indent everything within them!